addons/payment/tests/test_flows.py

```diff
@@ -1,6 +1,7 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from freezegun import freeze_time
+from urllib.parse import urlparse, parse_qs
```

Comment:
`urllib` is a stdlib.
```suggestion
from urllib.parse import urlparse, parse_qs

from freezegun import freeze_time
```